### PR TITLE
Feature assembly

### PR DESCRIPTION
**Pull request by davidlep**

### CODE REVIEW

File: app/__init__.py [hunk 1]
```diff
@@ -50,6 +52,8 @@ def after_request(response):
 api.add_namespace(realization_ns, path=BASE_URL + "/realization")
 api.add_namespace(realization_type_ns, path=BASE_URL + "/realizationType")
 api.add_namespace(employee_ns, path=BASE_URL + "/employee")
+api.add_namespace(assembly_ns, path=BASE_URL + '/assembly')
+api.add_namespace(assembly_type_ns, path=BASE_URL + '/assembly_type')
```
Comment (victortrinh) [inline app/__init__.py]: pascalCase

File: app/main/service/assembly_type_service.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    record.nameFr = data["nameFr"]
+    record.nameEn = data["nameEn"]
+    db.session.flush()
+    db.session.commit()
+    response_object = {
+        'status': 'success',
+        'message': 'Successfully updated row.',
+    }
+    return response_object, 201
+
```
Comment (SimonTrottier) [inline app/main/service/assembly_type_service.py:40]: Extra space

**Review by SimonTrottier — CHANGES_REQUESTED** [review]
need work

**Review by victortrinh — APPROVED** [review]

File: app/main/service/assembly_type_service.py (hunk 2 above)
Comment (victortrinh) [reply app/main/service/assembly_type_service.py:40]: It needs two spaces between functions tabarnak.

**Pull request merged**